common/src/test/java/com/microsoft/identity/common/internal/ui/webview/certbasedauth/TestNfcSmartcardCertBasedAuthManager.java

```diff
@@ -101,5 +101,12 @@ public void mockConnect(final boolean isChanged) {
 
     public void mockDisconnect() {
         mIsConnected = false;
+        if (mDisconnectionCallback != null) {
```

Comment:
what do we do is the callback is null? should we fail?

Reply:
same question for the usb counterpart

Reply:
We do nothing. The disconnection callback doesn't necessarily need to be set. For example, discovery for USB smartcards is always running while the auth WebView is showing, so a user could potentially plug in a smartcard, then unplug the smartcard, all without actually starting the CBA flow, and that would be an instance where we don't need to do anything upon disconnection.